napari/components/layerlist.py

```diff
@@ -186,6 +188,60 @@ def toggle_selected_visibility(self):
             if layer.selected:
                 layer.visible = not layer.visible
 
+    @property
+    def _extent_world(self):
+        """(2, D) array: Extent of layers in world coordinates.
```

Comment:
I'd really like to see us get away from this sort of type annotation (which can't be parsed by anything) if possible.  either a return type annotation (` -> np.ndarray`), and/or a numpy docstring `Returns` field since it's trickier to encode dimensionality at the moment in type annotations.

Reply:
@tlambert03 going to go with `def _extent_world(self) -> np.ndarray` is that what you had in mind? Agreed it would be great to standardize this stuff, but I can do it now for things I am adding new

Reply:
Actually, I can do both